skimage/_build.py

```diff
@@ -27,15 +27,19 @@ def cython(pyx_files, working_path=''):
     try:
         from Cython import __version__
         if LooseVersion(__version__) < '0.23':
-            raise RuntimeError('Cython >= 0.23 needed to build scikit-image')
+            raise ImportError
```

Comment:
I agree with @stefanv You can raise two different messages. Needs a little refactoring.